models/Challenge.js

```diff
@@ -2,6 +2,52 @@ const queries = require('./queries');
 const utils = require('../utils');
 
 class Challenge {
+  getAllChallengeablePeopleForUser(user) {
+    const sql = 'SELECT * FROM spots AS S, users AS U WHERE u.instrument = $1 AND u.part = $2 AND u.eligible = FALSE AND S.id = U.spotId';
```

Comment:
Maybe include performance here (or another query) so we don't have to make two queries